feeds/wlan-ap/opensync/src/platform/openwrt/src/netifd/src/wifi_inet_config.c

```diff
@@ -81,7 +81,10 @@ static void wifi_inet_conf_load(struct uci_section *s)
 
 	SCHEMA_SET_STR(conf.if_name, s->e.name);
 	if (tb[NET_ATTR_TYPE])
-		SCHEMA_SET_STR(conf.if_type, blobmsg_get_string(tb[NET_ATTR_TYPE]));
+                if ((strstr(s->e.name,"wan_") != NULL) || (strstr(s->e.name,"lan_") != NULL))
```

Comment:
Will osync Gateway always push vlan interface names as lan_x or wan_x irrespective of what user configures from the cloud?